native/main.ts

```diff
@@ -163,6 +164,18 @@ ipcMain.handle(RendererMessage.OPEN_URL, (_event, url) => {
   openExternalLink(url);
 });
 
+ipcMain.handle(RendererMessage.USERS_GIT_DEFAULT_BRANCH, () => {
+  try {
+    return childProcess
+      .execSync("git config --global --get init.defaultBranch", {
+        encoding: "utf-8",
+      })
```

Comment:
We should avoid sync calls so we don’t block the whole VM.

`child_process` doesn’t have great APIs for this so a possible solution could be this
```suggestion
    const stdout = await new Promise((resolve, reject) => 
      childProcess
      .exec("git config --global --get init.defaultBranch", {
        encoding: "utf-8",
      }, (error, stdout, stderr) => {
        if (error) {
          reject(error)
        } else {
          resolve(stdout)
        }
      })
```